howto/use-w3c-tracecontext/README.md

```diff
@@ -0,0 +1,243 @@
+# W3C Trace Context - Background
```

Comment:
Seems like "Background" should be the header for the first section not the entire article

Reply:
done.